app/_data/docs_nav_deck_1.12.x.yml

```diff
@@ -48,6 +48,8 @@ items:
         url: /guides/defaults
       - text: Using environment variables with decK
         url: /guides/environment-variables
+      - text: Managing decK with Docker
```

Comment:
Could we rename to `Run decK with Docker`? decK is the tool you use to manage Gateway configuration. You're not managing deck itself